<ui version="4.0" >
 <class>SvnLoginDialog</class>
 <widget class="QDialog" name="SvnLoginDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Subversion Login</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="3" column="0" colspan="2" >
      <widget class="QCheckBox" name="saveCheckBox" >
       <property name="toolTip" >
        <string>Select, if the login data should be saved.</string>
       </property>
       <property name="text" >
        <string>Save login data</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="passwordEdit" >
       <property name="toolTip" >
        <string>Enter password</string>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="realmLabel" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="usernameEdit" >
       <property name="toolTip" >
        <string>Enter username</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>usernameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>saveCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SvnLoginDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>23</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel" >
     <x>23</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SvnLoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>81</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel" >
     <x>81</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
